code/chart.js

```diff
@@ -2,3 +2,18 @@
 const ctx = document.getElementById('chart').getContext('2d')
 
 //"Draw" the chart here 👇
+
+const myChart = (numberOfProjects) => {
```

Comment:
This is a clear, short and simple code of the chart coded compared to the template they showed us from the site.  Easy to follow!